### ISSUE TITLE
[BUG]: Phalcon WebTools wrong migration directory

### ISSUE THREAD

**Issue by wajdijurry** [phalcon/phalcon-devtools#1438]
Hello,

I am using Phalcon WebTools to manage my app staff, it reads from "config.php" file where I specified all my app configs like modelsDir, controllersDir and migrationsDir, this is a part of my config.php file:
```
    'application' => [
        'modelsDir' => APP_PATH . '/common/models/',
        'controllersDir' => APP_PATH . '/modules/api/controllers/',
        'migrationsDir' => APP_PATH . '/migrations/',
        'logsDir' => APP_PATH . '/logs/',
        'uploadDir' => BASE_PATH . '/public/uploads/'
    ],
```
so when I came to Migrations view to create a new migration, it creates the migration file in a wrong directory, under root dir (/1.0.0), while it is supposed to create it under (/var/www/html/app/migrations) as specified in the form:
![Screenshot from 2020-03-29 20-32-33](https://user-images.githubusercontent.com/25797257/77855940-8842d500-71fc-11ea-9acf-3f22d23f73e8.png)

After digging into the code, I found a bug in Migrations script, which tries to read the migrations directory as array at first ```count($migrationsDirs) > 1```, then if not an array, it takes the first character of the string as the root path (else block, ```$migrationsDir = $migrationsDirs[0]```), so the result of "/var/www/html/app/migrations/" will be "/" which is wrong.
```
        if (count($migrationsDirs) > 1) {
            $question = 'Which migrations path would you like to use?' . PHP_EOL;
            foreach ($migrationsDirs as $id => $dir) {
                $question .= " [{$id}] $dir" . PHP_EOL;
            }
            fwrite(STDOUT, Color::info($question));
            $handle = fopen("php://stdin", "r");
            $line = (int)fgets($handle);
            if (!isset($migrationsDirs[$line])) {
                echo "ABORTING!\n";
                return false;
            }
            fclose($handle);
            $migrationsDir = $migrationsDirs[$line];
        } else {
            $migrationsDir = $migrationsDirs[0];
        }
```
### Expected and Actual Behavior

Expected:
```
/var/www/html/app/migrations/
```

Actual:
```
/
```

### Details

* System info and versions (if possible): (`phalcon info`)
* Phalcon Framework version: (`php --ri phalcon`)
phalcon

Web framework delivered as a C-extension for PHP
phalcon => enabled
Author => Phalcon Team and contributors
Version => 3.4.4
Build Date => Mar 29 2020 16:52:20
Powered by Zephir => Version 0.10.16-6826149172

Directive => Local Value => Master Value
phalcon.db.escape_identifiers => On => On
phalcon.db.force_casting => Off => Off
phalcon.orm.events => On => On
phalcon.orm.virtual_foreign_keys => On => On
phalcon.orm.column_renaming => On => On
phalcon.orm.not_null_validations => On => On
phalcon.orm.exception_on_failed_save => Off => Off
phalcon.orm.enable_literals => On => On
phalcon.orm.late_state_binding => Off => Off
phalcon.orm.enable_implicit_joins => On => On
phalcon.orm.cast_on_hydrate => Off => Off
phalcon.orm.ignore_unknown_columns => Off => Off
phalcon.orm.update_snapshot_on_save => On => On
phalcon.orm.disable_assign_setters => Off => Off

* PHP Version: (`php -v`)
PHP 7.3.15 (cli) (built: Feb 26 2020 12:46:49) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.15, Copyright (c) 1998-2018 Zend Technologies
    with Xdebug v2.9.4, Copyright (c) 2002-2020, by Derick Rethans

* Operating System: Ubuntu 19.10
* Server: Apache
* Other related info (Database, table schema): MySQL 8

**Comment by Jeckerson** [comment]
Hello

This is v3.4, which is deprecated. Do you really need it in v3.4? As it has been fixed in v4.

**Comment by wajdijurry** [comment]
Yes, I am still using v3.4, it is compatible with Phalcon v3.4.4, I did not upgrade to Phalcon 4 since it lacks many features and the most is ODM.

**Comment by Jeckerson** [comment]
Released in [v3.4.12](https://github.com/phalcon/phalcon-devtools/releases/tag/v3.4.12)

**Comment by wajdijurry** [comment]
Thank you!
